CHANGES.txt

```diff
@@ -7,6 +7,20 @@ Features
 * Delete old ``bootstrap`` theme (use ``bootstrap3`` instead)
 * Screen reader-friendly navbar collapses and dropdowns (Issue #1863)
 * Modern reST stylesheets, based in part on Bootstrap 3 (Issue #1150)
+* Several improvements to WordPress importer (PR #1867):
+  * Allowing to export categories and category hierarchy with
+    --export-categories-as-categories
+  * Allowing to exclude private posts, and allowing to include empty posts
+  * Allowing to use HTTP authentication for downloads with --download-auth
+  * Allowing to export comments with --export-comments
+  * Allowing to use WordPress page compiler to directly convert posts
+    to HTML on import with --transform-to-html
+  * Allowing to use WordPress page compiler on imported site instead of
+    converting posts to markdown with --use-wordpress-compiler
+  * Allowing to automatically install the WordPress page compiler when
+    needed with --install_wordpress_compiler
```

Comment:
use `-` instead of `_`
also, you should advertise the compiler if the user does not ask for it